sources/ubiome.py

```diff
@@ -90,59 +90,71 @@ def create_datafiles(username, samples=None, task_id=None, update_url=None,
     if not samples:
         raise Exception('`samples` parameter missing')
 
+    # Number samples if there is more than one.
+    sample_i = 0
+    if len(samples) > 1:
+        sample_i = 1
+
     for sample in samples:
```

Comment:
possible to use `enumerate` here like `for i, sample in enumerate(samples)` (then your check below becomes `str(i) if len(samples) > 1`)